lib/razorpay/invoice.rb

```diff
@@ -20,5 +20,43 @@ def self.fetch(id)
     def self.all(options = {})
       request.all options
     end
+
+    def self.update(id, data = {})
+      request.patch id, data
+    end
+
+    def self.issue(id)
+      request.post "#{id}/issue"
+    end
+
+    def self.cancel(id)
+      request.post "#{id}/cancel"
+    end
+
+    def update(data = {})
+      self.class.update id, data
+    end
+
+    def issue
+      self.class.issue(id)
+    end
+
+    def cancel
+      self.class.cancel(id)
```

Comment:
Call functions without arguments in parantheses please

Reply:
Done :white_check_mark: